examples/apps/table-detail-tea/src/main.rs

```diff
@@ -0,0 +1,239 @@
+use std::io::stdout;
+/// A Ratatui example that demonstrates the Elm architecture with a basic list - detail
+/// application.
+///
+/// This example runs with the Ratatui library code in the branch that you are currently
+/// reading. See the [`latest`] branch for the code which works with the most recent Ratatui
+/// release.
+///
+/// [`latest`]: https://github.com/ratatui/ratatui/tree/latest
+use std::time::Duration;
+
+use crossterm::event::{
+    self, DisableMouseCapture, EnableMouseCapture, Event, KeyCode, MouseEventKind,
+};
+use crossterm::ExecutableCommand;
+use ratatui::layout::{Constraint, Layout};
+use ratatui::style::{Modifier, Style};
+use ratatui::text::Line;
+use ratatui::widgets::{Block, Borders, Padding, Paragraph, Row, Table, TableState};
+use ratatui::Frame;
+
+static INFO_TEXT: &str = " (Esc/q) quit | (↑) move up | (↓) move down ";
+
+/// Application data model and state
+#[derive(Debug, Default)]
+struct Model {
+    table_items: Vec<Data>,
+    table_state: TableState,
+    running_state: RunningState,
+}
+
+#[derive(Debug, Default, PartialEq, Eq)]
+enum RunningState {
+    #[default]
+    Running,
+    Done,
+}
+
+/// Data model
+#[derive(Debug, Default)]
+struct Data {
+    name: String,
+    address: String,
+    email: String,
+}
+
+#[derive(PartialEq, Copy, Clone)]
+enum Message {
+    SelectNext,
+    SelectPrevious,
+    Quit,
+}
+
+fn main() -> color_eyre::Result<()> {
+    tui::install_panic_hook();
+    let mut terminal = tui::init_terminal()?;
```

Comment:
use `ratatui::init()` here instead of having a full tui module
For disabling mouse capture, have a small method which just handles resetting the mouse capture, it will run before the ratatui installed hook